src/js_realm.hpp

```diff
@@ -445,12 +445,12 @@ class RealmClass : public ClassDefinition<T, SharedRealm, ObservableClass<T>> {
     };
 
     PropertyMap<T> const properties = {
-        {"empty", {wrap<get_empty>, nullptr}},
+        {"isEmpty", {wrap<get_empty>, nullptr}},
         {"path", {wrap<get_path>, nullptr}},
         {"schemaVersion", {wrap<get_schema_version>, nullptr}},
         {"schema", {wrap<get_schema>, nullptr}},
         {"inMemory", {wrap<get_in_memory>, nullptr}},
-        {"readOnly", {wrap<get_read_only>, nullptr}},
+        {"isReadOnly", {wrap<get_read_only>, nullptr}},
         {"isInTransaction", {wrap<get_is_in_transaction>, nullptr}},
```

Comment:
I wonder if we want to align this with `inMemory` too? Either rename `inMemory` to `isInMemory` rename this to `inTransaction`.

Reply:
I was about to...then I realized this is undocumented.  I'm not even sure if we should be exposing this and what purpose it serves

Reply:
It's documented? https://www.mongodb.com/docs/realm-sdks/js/latest/Realm.html#isInTransaction

Reply:
Ahh you meant `inMemory` :)